### DISCUSSION TITLE
Microsoft Access Connection

### DISCUSSION THREAD

**Discussion by miquelmartorell** [apache/airflow#39864] [Q&A]
Good morning,

I need to create a DAG to connect to a Microsoft Access database (yes, in 2024 we still have a small program using Microsoft Access) from Airflow running in a Docker container on an Ubuntu Server. 

I have tried using the mdbtools library (https://github.com/mdbtools/mdbtools), following the precise installation and configuration instructions inside the Docker container, but when I run the DAG I always get a pyodbc.Error: ('HY000', 'The driver did not supply an error!'). 

Has anyone had the need to connect to a Microsoft Access database? If so, how did you do it? 

Thank you very much for your appreciated help.

**Comment by HongVietLE** [comment]
Using a connection string and the ODBC driver, I think it could do the job

**Comment by mathmorfose** [comment]
@miquelmartorell were you able to solve it?
